tools/src/tester/StoryEvaluator.ts

```diff
@@ -8,11 +8,11 @@
 */
 
 import { type Chapter, type Story, type SupplementalChapter } from './types/story.types'
-import { type ChapterEvaluation, Result, type StoryEvaluation } from './types/eval.types'
+import { type ChapterEvaluation, Result, type StoryEvaluation, ChaptersEvaluations, StoryOutputs, Evaluation } from './types/eval.types'
 import ChapterEvaluator from './ChapterEvaluator'
 import type ChapterReader from './ChapterReader'
 import SharedResources from './SharedResources'
-import { overall_result } from './helpers'
+import { check_story_variables, extract_output_values, overall_result } from './helpers'
```

Comment:
If `check_story_variables` is only used here, it should be defined here instead.